compiler/rustc_data_structures/Cargo.toml

```diff
@@ -28,6 +25,10 @@ libc = "0.2"
 stacker = "0.1.14"
 tempfile = "3.2"
 
+indexmap = { version = "1.8.0", optional = true }
```

Comment:
This shouldn't be optional because we always use it -- only the rustc-rayon feature is conditional.